src/language/parser.js

```diff
@@ -117,6 +117,19 @@ export type ParseOptions = {
    * disables that behavior for performance or testing.
    */
   noLocation?: boolean,
+
+  /**
+   * If enabled, the parser will understand and parse variable definitions
+   * contained in a fragment definition. They'll be represented in the
+   * `variableDefinitions` field of the FragmentDefinitionNode.
+   *
+   * The syntax is identical to normal, query-defined variables. For example:
+   *
+   *   fragment A on T ($var: Boolean = false) {
+   *     ...
+   *   }
+   */
+  fragmentVariablesExperimental?: boolean,
```

Comment:
This option reads a bit strangely, what about `experimentalFragmentVariables`?

Reply:
Yeah, I had `fragmentVariables_experimental` before (not sure if that was any better), but the camelcase linter didn't like that. I'll take your suggestion.